docs/configuration/conflict_resolution.rst

```diff
@@ -0,0 +1,27 @@
+.. include:: ../global.rst.inc
+
+
+Conflict Resolution Options
+===========================
+
+
+::
+
+   jobs:
+   - type: push
+     filesystems: ...
+     conflict_resolution:
+       initial_replication:
+           send_all_snapshots: false # false,true
```

Comment:
We need to iterate on this, since, as suggested in https://github.com/zrepl/zrepl/issues/408#issuecomment-795329846, there are three possible options:

* fail
* most recent
* all

I think we should add the `fail` policy option to this PR as well. The benefit will be that we're forced to define a config syntax that deals with non-boolean policy options, leading to less breaking changes down the road.
I'm not convinced that the `policy: { type: ... }` schema in #408 is a good syntax (too verbose), feel free to come up with a better one. (The advantage of `type: ` is that our config parser has infrastructure to handle this kind of 'enum').

Reply:
Yeah, I wanted to go this route but I didn't quite understand how to do it. I ran out of time to pursue this further right now, though. I'm not sure but I think there might also be a bug in the scenario where the first replication of many snapshots gets interrupted and isn't resumed properly. Sorry for dropping it here, I could close it if you would like.